components/esp_dns/README.md

```diff
@@ -0,0 +1,189 @@
+| Supported Targets | ESP32 | ESP32-C2 | ESP32-C3 | ESP32-C5 | ESP32-C6 | ESP32-C61 | ESP32-H2 | ESP32-P4 | ESP32-S2 | ESP32-S3 |
+| ----------------- | ----- | -------- | -------- | -------- | -------- | --------- | -------- | -------- | -------- | -------- |
```

Comment:
we usually place this in example README's -- I'd suggest removing from component's docs as this DNS module should support all targets

Reply:
Done